packages/@aws-cdk/aws-cloudformation/test/integ.core-custom-resources.ts

```diff
@@ -0,0 +1,31 @@
+import { App, CfnOutput, Construct, CustomResource, CustomResourceProvider, CustomResourceProviderRuntime, Stack, Token } from '@aws-cdk/core';
+
+class TestStack extends Stack {
```

Comment:
Would it make sense to put a file in core with the same (or similar) name but with the content

```
// Since cdk-integ depends on cdk which depends on @aws-cdk/core
// (as a "dev dependency"), this integ test has been added to the package 
// @aws-cdk/aws-cloudformation.
```

Reply:
done. Good idea

Reply:
Actually, I can't call this "integ." because pkglint goes haywire and requires that this package will have a full integration test setup...

Reply:
Called this `integ.core-custom-resources.readme.txt`